### DISCUSSION TITLE
Some problems

### DISCUSSION THREAD

**Discussion by Zingel1986** [dbuezas/lovelace-plotly-graph-card#12] [General]
Hi,

Maybe I get something wrong, but to use this repo don´t you need to have a repository.json/yaml at the root here? (https://developers.home-assistant.io/docs/add-ons/repository/)
How can I add the repo to my HA, the md says only add repo address... 

thanks in advance
Zingel

OK, forget it. I did not know that I had to install HACS separately. Its installed now. I only have problems to get it working in a card. I always get the error "Custom element not found: plotly-graph"

**Comment by Zingel1986** [comment]
OK, I got the plotly card now. But the data points are not updated when I am panning or zooming outside of the initial time frame as shown in your first gif in the readme.md
What am I doing wrong here?

**Reply by Zingel1986** [reply]
It has started working correctly on its own.

**Reply by dbuezas** [reply]
Ok, thanks for letting me know.
I recently pushed a patch that may be related. Make sure to update it :)
